EmbedVideo.hooks.php

```diff
@@ -72,16 +72,15 @@ class EmbedVideoHooks {
 	 * @return void
 	 */
 	public static function onExtension() {
-		global $wgEmbedVideoDefaultWidth, $wgMediaHandlers, $wgFileExtensions;
-
-		$config = MediaWikiServices::getInstance()->getConfigFactory()->makeConfig('main');
+		global $wgEmbedVideoDefaultWidth, $wgMediaHandlers, $wgFileExtensions,
+			$wgEmbedVideoEnableAudioHandler, $wgEmbedVideoEnableVideoHandler, $wgEmbedVideoAddFileExtensions;
```

Comment:
Lets get all those values from `MainConfig`. If possible, rewrite the hook to a hook handler https://www.mediawiki.org/wiki/Manual:Hooks

Reply:
Ok

Reply:
Not possible to get these values from MainConfig as calling MediaWikiServices here is deprecated and function must be static to call in extension registration so dependency injection also is unavailable and it's impossible to change configuration values with MainConfig. In Wikimedia extensions these configurations variables are also used by global. Example: (https://github.com/wikimedia/mediawiki-extensions-CirrusSearch/blob/master/includes/Hooks.php)